src/components/pages/order/OrderPage.js

```diff
@@ -1,27 +1,30 @@
-import { useState } from 'react';
+import { useRef, useState } from 'react';
 import styled from 'styled-components';
 import OrderContext from '../../../context/OrderContext';
 import { theme } from '../../../theme';
 import Main from './main/Main';
 import Navbar from './navbar/Navbar';
 import { fakeMenu } from "../../../data/fakeMenu";
-import { EMPTY_PRODUCT } from './main/Admin/AdminPanel/AddProductForm';
+import { EMPTY_PRODUCT } from "../../../enums/product";
+import { deepClone } from '../../../utils/array';
 
 
 
 export default function OrderPage(){
   // state 
   const [isModeAdmin, setIsModeAdmin] = useState(false);
   const [isCollapsed, setIsCollapsed] = useState(false)
-  const [currentTabSelected, setcurrentTabSelected] = useState("add")
+  const [currentTabSelected, setCurrentTabSelected] = useState("add")
   const [menu, setMenu] = useState(fakeMenu.MEDIUM)
   const [newProduct, setNewProduct] = useState(EMPTY_PRODUCT)
+  const [productSelected, setProductSelected] = useState(EMPTY_PRODUCT)
+  const titleEditRef = useRef()
 
 
   // comportements
   const handleAddProduct = (newProduct) => {
```

Comment:
```suggestion
  const handleAddMenu = (newProduct) => {
```

On **sait** que le basket va arriver quelques tickets plus tard. Change le maintenant. Ca t'évitera de tout changer après. Comme on le "sait", ce n'est pas du YAGNI.

Pense à le changer pour les autres, handleEditMenu, handleDeleteMenu, handleAddMenu, etc.